src/Quadmath.jl

```diff
@@ -15,9 +15,11 @@ import Base: (*), +, -, /,  <, <=, ==, ^, convert,
           copysign, flipsign, max, min, hypot, abs,
           ldexp, frexp, modf, nextfloat, eps,
           isinf, isnan, isfinite, isinteger,
-          floatmin, floatmax, precision, signbit,
+          floatmin, floatmax, precision, signbit, maxintfloat,
           Int32, Int64, Float64, BigFloat, BigInt
 
+using Random
```

Comment:
I was actually thinking we should also do the same for `Printf`: since it is a stdlib, we probably don't need Requires.jl for it.

Reply:
We're using Requires for SpecialFunctions, not Printf.

I have also been confused about the Printf package;  at present the methods imported and extended here are all in Base.Printf (only the user-facing macros are in the Printf stdlib).

Reply:
Ah, I had assumed it was using Requires because it wasn't in the Project.toml. That makes sense.